### PR TITLE
8701 - changes for renewal

### PR DESCRIPTION
**Pull request by jordiwes**
*Issue #:* /bcgov/entity#8701

*Description of changes:*
- add fee summary
- additional rules for renew

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/composables/fees/factories/useFeeSummary.ts [hunk 1]
```diff
@@ -45,8 +45,20 @@ export function getFeeSummary (
     return { ...defaultFeeSummaries[FeeSummaryDefaults.NO_FEE] }
   }
   if (feeType === FeeSummaryTypes.RENEW) {
```
Comment (kialj876) [inline ppr-ui/src/composables/fees/factories/useFeeSummary.ts]: hey Steph, can this just be combined into the FeeSummaryTypes.NEW block? ```if ([FeeSummaryTypes.RENEW, FeeSummaryTypes.NEW].includes(feeType)) ....```

**Review by kialj876 — CHANGES_REQUESTED** [review]
Looks good, just one comment and feeSummary needs a test suite update for renewals! Also the label probably needs to change (right now it is under a FUTURE comment and is defaulting to 'Renewal')

**Review by doug-lovett — COMMENTED** [review]
Nothing to add in addition to Kial's review.

**Review by doug-lovett — APPROVED** [review]

**Comment by kialj876** [comment]
Hey Steph, I think you'll need to add renewal to the condition on line 80 in useFeeSummary (and that should fix the test failure too) - assuming the hints follow the same logic as a new registration

**Comment by jordiwes** [comment]
Thanks Kial, I made those changes

**Pull request merged**